ibm/mas_devops/roles/bas_install/templates/bascfg.yml.j2

```diff
@@ -30,6 +30,42 @@ spec:
       secretName: bas-apikey
     segmentKey: "{{bas_segment_key}}"
   certificates: 
-    - alias: ca
+    - alias: part1
       crt: |
-        {{bas_tls_crt | indent(8) }}
\ No newline at end of file
+        {{bas_tls_crt[0] | indent(8) }}
+    - alias: part2
+      crt: |
+        {{bas_tls_crt[1] | indent(8) }}
+    - alias: part3
+      crt: |
+        -----BEGIN CERTIFICATE-----
```

Comment:
Is it OK to include this certificate in our code? I see that is has come from an existing public source: [isrgrootx1.pem](https://letsencrypt.org/certs/isrgrootx1.pem) but can you confirm that we are able to include this in our code or should we dynamically include it at script run-time?

Reply:
i would think it should be okay as it is publicly available: https://letsencrypt.org/certs/isrgrootx1.pem
whether we want to hardcode it or include it via another way... I don't know.
@durera @CAIOFCP any concerns with this approach?